### ISSUE TITLE
Client Disconnected after using Recall

### ISSUE THREAD

**Issue by 77697a65** [Lady-Binary/EasyLOU#119]
**Describe the bug**
When I recall from a far distance, once I arrive at destination, Client disconnects.

**To Reproduce**

Steps to reproduce the behavior. 

For example:

```
repeat		
	print("recalling bank..")	
	FindItem("Bank", BACKPACKID)
	runeID = FINDITEM[1].ID
	ContextMenu(runeID, "Recall")	
	sleep(5000)
	myx = CHARPOSX	
until myx == 1813.577
```

Log:

```
2020-11-17T22:10:16.8596232Z - LOU - Error updating status: System.NullReferenceException
  at (wrapper managed-to-native) UnityEngine.Component.get_gameObject(UnityEngine.Component)
  at UnityEngine.Component.GetComponentInChildren (System.Type t, System.Boolean includeInactive) [0x00001] in <548b4fa0e7e04f27a1b7580930bfb7dc>:0 
  at UnityEngine.Component.GetComponentInChildren[T] () [0x00001] in <548b4fa0e7e04f27a1b7580930bfb7dc>:0 
  at LOU.Worker.<ProcessClientCommand>b__37_2 (System.Collections.Generic.KeyValuePair`2[TKey,TValue] f) [0x00011] in <4a4cb5dad14242ceb65bd7e87fe5ed92>:0 
  at (wrapper delegate-invoke) System.Func`2[System.Collections.Generic.KeyValuePair`2[System.String,ClientObject],LOU.ClientStatus+FINDPERMANENTStruct].invoke_TResult_T(System.Collections.Generic.KeyValuePair`2<string, ClientObject>)
  at System.Linq.Enumerable+SelectEnumerableIterator`2[TSource,TResult].ToArray () [0x0001d] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.Buffer`1[TElement]..ctor (System.Collections.Generic.IEnumerable`1[T] source) [0x0000a] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.OrderedEnumerable`1[TElement].ToArray () [0x00000] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at System.Linq.Enumerable.ToArray[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0001f] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at LOU.Worker.UpdateClientStatus () [0x00993] in <4a4cb5dad14242ceb65bd7e87fe5ed92>:0 
  at LOU.Worker.Update () [0x00210] in <4a4cb5dad14242ceb65bd7e87fe5ed92>:0  
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)
```

**Comment by 77697a65** [comment]
If anyone encounters this `ResetVars()` resolves this problem.

**Comment by Lady-Binary** [comment]
I believe the fix for #120 should also fix this one, please let me know how it goes with the latest version 1.0.0.7 @77697a65
